itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/txn/compactor/TestCompactor.java

```diff
@@ -1165,7 +1165,7 @@ public void testTableProperties() throws Exception {
         "'transactional'='true'," +
         "'compactor.mapreduce.map.memory.mb'='2048'," + // 2048 MB memory for compaction map job
         "'compactorthreshold.hive.compactor.delta.num.threshold'='4'," +  // minor compaction if more than 4 delta dirs
-        "'compactorthreshold.hive.compactor.delta.pct.threshold'='0.5'" + // major compaction if more than 50%
```

Comment:
This is because the file sizes changed a little. The delta was 49.7% of the base, so it wasn't compacting. Changing the percentage to 49% gets the compactor to still compact the table.